app/components/task/holder.hbs

```diff
@@ -20,25 +20,23 @@
       />
     {{/if}}
 
-    {{#if (not-eq @task.status this.TASK_KEYS.VERIFIED)}}
+    {{#if (not-eq this.status this.TASK_KEYS.VERIFIED)}}
       <div class='task-update-container'>
         <label id='task-update-label' for='task-update'><b>Status:</b></label>
         <select
+          data-test-task-status-select
           id='task-update'
           {{on 'change' this.onStatusChange}}
-          {{on 'change' (fn @onTaskUpdate @task.id)}}
+          {{on 'change' (fn this.onUpdate @task.id)}}
         >
           {{#each this.availabletaskStatusList as |taskStatus|}}
             {{#if (not-eq taskStatus.key this.TASK_KEYS.ALL)}}
-              {{#if (eq taskStatus.key @task.status)}}
-                <option value={{taskStatus.key}} selected>
-                  {{taskStatus.displayLabel}}
-                </option>
-              {{else}}
-                <option value={{taskStatus.key}}>
-                  {{taskStatus.displayLabel}}
-                </option>
-              {{/if}}
+              <option
+                value={{taskStatus.key}}
+                selected={{eq taskStatus.key this.status}}
+              >
+                {{taskStatus.displayLabel}}
+              </option>
```

Comment:
It was previously added by me while updating the tasks card design. Felt it could be improved here. I have updated the description to mention the refactor. 

FYR: https://github.com/Real-Dev-Squad/website-my/pull/420/files#diff-0b9a6a0064f4b9f4c04ec7d80be0221b03aa7bc55fe0682d4ddc5f403aea176cR33.

Reply:
How did you improve it here?

Reply:
Previously we were unnecessarily repeating the whole `option` element twice. As the only difference was the `selected` attribute this would be more concise.

Reply:
Got it